ReactAndroid/src/main/jni/react/jni/NativeTime.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <time.h>
+#include "NativeTime.h"
+
+namespace facebook {
+namespace react {
+
+double reactAndroidNativePerformanceNowHook() {
+  struct timespec now;
+  clock_gettime(CLOCK_MONOTONIC, &now);
+
+  return (now.tv_sec * 1000000000LL + now.tv_nsec) / 1000000.0;
```

Comment:
nit: I think it'd be preferable to create more semantic consts for these values (e.g. NANOSECONDS_IN_SECOND) so that it's easier for people to understand exactly what this is doing

Reply:
Sure, I moved it to a constant, thanks!